manifests/rule.pp

```diff
@@ -25,7 +25,7 @@
 #     rule =>  "if prifilt('*.info') then stop"
 #   }
 #
-# @param name [Pattern['^[^/]\S+/\S+\.conf$']]
+# @attr name [Pattern['^[^/]\S+/\S+\.conf$']]
```

Comment:
This won't show up in the final documentation.

See: https://tickets.puppetlabs.com/browse/PDOC-153